### PR TITLE
Move generic documentation to shared location

### PR DESCRIPTION
**Pull request by adrienbernede**
This PR moves and complete documentation to a shared location (versioned in https://github.com/LLNL/radiuss-ci).

The parts of the documentation concerned are the usage of Uberenv and Gitlab CI.

Relates to https://github.com/LLNL/Umpire/pull/442 and https://github.com/LLNL/CHAI/pull/159

### CODE REVIEW

**Comment by adrienbernede** [comment]
LGTM

**Comment by adrienbernede** [comment]
@davidbeckingsale It seems to me the changes here are harmless enough to be merged ;)

**Comment by adrienbernede** [comment]
@davidbeckingsale will it merged this time? ;)

**Comment by adrienbernede** [comment]
@rhornung67 I would appreciate some feedback here: similar changes were merged in Umpire and CHAI.
Anything left to do to get it ready for RAJA too?

File: docs/sphinx/dev_guide/host_config.rst [hunk 1]
```diff
@@ -1,12 +1,4 @@
-.. ##
-.. ## Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
-.. ## and RAJA project contributors. See the RAJA/COPYRIGHT file
-.. ## for details.
-.. ##
-.. ## SPDX-License-Identifier: (BSD-3-Clause)
-.. ##
-
-.. raja_host_config:
+.. host_config:
```
Comment (rhornung67) [inline docs/sphinx/dev_guide/host_config.rst:1]: Please don't remove the copyright statement from files.

Also, please add it to the other files you added.

**Review by rhornung67 — CHANGES_REQUESTED** [review]
Looks OK to me, except for my comments on the copyright statements.  Also, branch needs to be updated with base branch.

**Comment by adrienbernede** [comment]
@rhornung67 Done. Thank you!

**Review by rhornung67 — APPROVED** [review]

**Comment by adrienbernede** [comment]
@rhornung67 Should I motivate another reviewer or is it OK like this (I don't have merge powers)?

**Comment by rhornung67** [comment]
@davidbeckingsale when you have a minute, please look this over. If you're OK with it, we'll merge i t.

**Comment by davidbeckingsale** [comment]
Why does this have a BLT change?

**Comment by rhornung67** [comment]
> Why does this have a BLT change?

I was wondering that too. That's why I wanted you to look at this.

**Comment by adrienbernede** [comment]
Hmm, maybe a mix-up during the merge conflict resolution... I’ll have a look.

**Comment by adrienbernede** [comment]
@rhornung67 @davidbeckingsale Sorry for overlooking this... That was indeed a mistake after a merge.

**Review by davidbeckingsale — APPROVED** [review]

**Review by rhornung67 — APPROVED** [review]

**Pull request merged**